### ISSUE TITLE
CT: This Disposable has already been disposed, and cannot be disposed again

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/balancing-act#154]
Probably introduced in https://github.com/phetsims/balancing-act/commit/e548ffd49532424fe0f132a12c9c614d2664b54b for https://github.com/phetsims/balancing-act/issues/94.  And probably the cause of https://github.com/phetsims/balancing-act/issues/153.

```
balancing-act : pan-and-zoom-fuzz : unbuilt
http://128.138.93.172/continuous-testing/ct-snapshots/1710169074428/balancing-act/balancing-act_en.html?continuousTest=%7B%22test%22%3A%5B%22balancing-act%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1710169074428%22%2C%22timestamp%22%3A1710169473338%7D&brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=true
Query: brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=true
Uncaught Error: Assertion failed: This Disposable has already been disposed, and cannot be disposed again
Error: Assertion failed: This Disposable has already been disposed, and cannot be disposed again
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1710169074428/assert/js/assert.js:28:13)
at assert (Disposable.ts:49:18)
at dispose (BalanceGameView.js:150:13)
at listener (TinyEmitter.ts:123:8)
at emit (Emitter.ts:62:21)
at emit (createObservableArray.ts:314:57)
at pop (createObservableArray.ts:399:46)
at clear (BalanceGameModel.js:171:23)
at setChallenge (BalanceGameModel.js:146:9)
at startLevel (BalanceGameView.js:170:18)
```

**Comment by jbphet** [comment]
In an effort to fix the memory leaks described in #94 I had added disposal of masses when they are removed from the `BalanceGameModel`.  However, these masses are sometimes removed and then added *back* to the model to show the correct answer if the user never got the challenge correct.  So disposing them when removed from the model won't work.  They should be disposed with the game challenge, since that's where they are created.  I'll do that as part of the work for #94, and for now have just removed the `dispose()` call that is causing this error.

I'll leave this open until I'm not seeing the problem on CT anymore.

**Comment by jbphet** [comment]
CT has been clear for several rounds now - closing.
